fdbclient/FDBTypes.h

```diff
@@ -648,4 +648,72 @@ struct ClusterControllerPriorityInfo {
 	}
 };
 
+struct HealthMetrics {
+	struct StorageStats {
+		int64_t storageQueue;
+		int64_t storageNDV;
```

Comment:
We call this durability lag in status (measured in both versions and seconds), and it might be good to keep the nomenclature consistent. Also given the above, the expansion of the NDV acronym may not be widely used or known.

Reply:
It actually looks like durability lag is the difference between the newest version and the durable version on the storage server. The NDV is the difference between the desired oldest version and what is actually durable. As far as I can tell, this is something we have never monitored before.

It seems like it would be a good measure for if the updateStorage actor is not getting enough CPU to make versions durable, or if there is some sort of disk issue that is preventing the storage server from making versions durable.

Reply:
Interesting. Is this a substantive difference for the purposes we are using it for? We also have data lag which when combined with the data version should be pretty similar, but it lagging represents a different sort of problem than the durable version lagging.

Reply:
Oh, I think I misread what you said and got confused about which metric was measuring what. 

I think in practice, we use durability lag to measure the same thing as what's being measured here, even if the specifics aren't exactly the same.